conf/locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -26,8 +26,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: 0.1a\n"
 "Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n"
-"POT-Creation-Date: 2017-03-30 10:52+0000\n"
-"PO-Revision-Date: 2017-03-30 10:54:40.500506\n"
+"POT-Creation-Date: 2017-04-04 04:13+0000\n"
```

Comment:
Please remove all the *.po files from this PR. Translations are updated through Transafix, and periodically checked into edx-platform via translation-only commits. We don't hand-edit these files.